.github/workflows/setup-latest.yml

```diff
@@ -44,9 +44,9 @@ jobs:
         os:
           [
             ubuntu-22.04,
-            ubuntu-20.04,
-            macos-12,
+            macos-13,
             macos-14,
-            windows-2019,
-            windows-2022
+            windows-2022,
```

Comment:
ditto

Reply:
That... should certainly be windows?

Reply:
Ah, I couldn't see the duplication here.

Reply:
I've added macOS-15 for good measure.